kong/db/schema/others/declarative_config.lua

```diff
@@ -506,6 +507,71 @@ local function validate_references(self, input)
 end
 
 
+function DeclarativeConfig.validate_references_full(self, input)
```

Comment:
Why the parameter is call `self`? I think that `DeclarativeConfig` has no method `new()` to create instance.

Reply:
validate_references needs self. this function just tries to export validate_references